src/linux/cgroups2.hpp

```diff
@@ -69,6 +71,20 @@ Try<std::set<std::string>> enabled(const std::string& cgroup);
 
 } // namespace controllers {
 
+namespace devices {
+
+using cgroups::devices::Entry;
+
+// Configure the device access permissions for the cgroup. These permissions
+// are hierarchical. I.e. if a parent cgroup does not allow an access then
+// 'this' cgroup will be denied access.
+Try<Nothing> configure(
+    const std::string& cgroup,
+    const std::vector<Entry> allow,
+    const std::vector<Entry> deny);
```

Comment:
We typically do not take a const non reference like this. The options are:

* const vector& (if we don't need to mutate it / store it)
* vector&& (if we want to force a move to avoid copying)
* vector (if we need to mutate it / store a mutated version of it, and want to support both move and const ref / copy)

Reply:
Will update to be `const vector&` :+1: